src/dewey/curation.clj

```diff
@@ -60,6 +60,10 @@
     (when (or (= type :data-object) (indexable? entity)) (op entity))
     (indexing/remove-entity es entity-type entity-id)))
 
+(defn- apply-or-remove-or-noop
```

Comment:
This function, and the changes within this file, are generally about contacting elasticsearch as a first step, and no-oping if the thing in question isn't indexed.